src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -92,13 +92,14 @@ class FormPage extends React.Component {
       : this.props.form.data;
   };
 
-  goBack = () => {
+  goBack = (customPath = '') => {
     const {
       form,
       route: { pageList },
       location,
     } = this.props;
-    const path = getPreviousPagePath(pageList, form.data, location.pathname);
+    const path =
+      customPath || getPreviousPagePath(pageList, form.data, location.pathname);
```

Comment:
I'm a little concerned about this implementation, just because there are no checks on whether the customPath is valid. I know in your documentation PR you note that "invalid paths will return you to the root (introduction) page", but I don't see any logic here that looks like that redirect happens. I could be missing it somewhere (I'm not as familiar with the forms system yet), but since the `customPath` is bypassing the part of `getPreviousPagePath` that manages that redirect, I'm not sure how it would work. 

Would it be possible to implement this in a way that the custom path gets checked by the [`getEligiblePages` ](https://github.com/department-of-veterans-affairs/vets-website/blob/6f7261eec3927c573427c3099a617d053697ff7f/src/platform/forms-system/src/js/routing.js#L15)function and is routed appropriately if the custom path is not eligible?

Reply:
Ahh, yes, the return to the root (introduction) page is the default behavior of the router. And yes, I think that would be a good idea to make sure the path is valid. I'll update it to check.